Add OIDC provider docs for IBM ISAM

godvoigt opened the pull request:
This adds the docs for the IBM ISAM OIDC provider

austingebauer commented:
Thanks for this, @godvoigt! Are you able to sign the CLA?

austingebauer commented:
This is looking good, but I think it's missing a few things. Could you add changes similar to what was done in https://github.com/hashicorp/vault/pull/16274/files?

godvoigt commented:
Hi @austingebauer, I will take a look at your suggestion.

As for the CLA, it's the same as for https://github.com/hashicorp/vault-plugin-auth-jwt/pull/222. I signed it, but the state is not updated here. Clicking on "Refresh it" only takes me to my Github profile.

godvoigt commented:
@austingebauer I've added the files you requested similar to the SecureAuth implementation. I also fixed the CLA now (my company mail changed a while ago and I hadn't configured / verified the new one in GitHub)

austingebauer reviewed (approved):
Thanks, @godvoigt!
